package.json

```diff
@@ -15,14 +15,19 @@
         "@openzeppelin/contracts": "^4.3.2",
         "@openzeppelin/hardhat-upgrades": "^1.9.0",
         "@openzeppelin/truffle-upgrades": "^1.3.1",
-        "@types/node": "^17.0.8",
+        "@typechain/ethers-v5": "^9.0.0",
+        "@typechain/hardhat": "^4.0.0",
+        "@types/chai": "^4.3.0",
+        "@types/mocha": "^9.1.0",
+        "@types/node": "^17.0.13",
```

Comment:
I added the chain and mocha types to allow the tests to understand the keywords mocha defines (descirbe, etc) - its also [recommended by hardhat](https://hardhat.org/guides/typescript.html#installing-dependencies)